t2.cpp

```diff
@@ -0,0 +1,6 @@
+sdjfsjsdjfsd;
```

Comment:
variable 'sdjfsjsdjfsd' is non-const and globally accessible, consider making it const [cppcoreguidelines-avoid-non-const-global-variables]